README.md

```diff
@@ -54,9 +49,59 @@ async function handleEvent(event) {
 }
 ```
 
+### Optional Arguments
+
+You can customize the behavior of `getAssetFromKV` by passing the following properties as an object into the second argument 
+
+```
+return getAssetFromKV(event, { mapRequestToAsset: ... })
+```
+
+#### `mapRequestToAsset`
+
+type: function(Request) => Request
```

Comment:
```suggestion
type: (req: Request) => Request
```
nit TS syntax

Reply:
is not TS yet

Reply:
just more proper for any JS project ( I had it wrong at first too) just a nit.

Reply:
hrm. is this even a, like, "normal" documentation syntax?